### ISSUE TITLE
Issues after Cloud connector update

### ISSUE THREAD

**Issue by samaanbag** [sacha81/XA-and-XD-HealthCheck#79]
The script no longer works after a recent cloud connector upgrade.

When the script runs now we get the following message:

Connector Machine detected. SDK's cmdlet are not supported on Cloud Connectors.

Are there any plans to update the script in the near future?

Thanks
Samaan
![Screenshot_1](https://github.com/sacha81/XA-and-XD-HealthCheck/assets/162441148/933f6285-dcd3-4f9c-8f80-d299733b9b5c)

**Comment by sacha81** [comment]
No, I will release a new script specific for Citrix Cloud based on REST API

**Comment by sacha81** [comment]
I've started a new repo and added my current personal used version which is far away from perfect  but working for me because of the currently urgent need:  https://github.com/sacha81/citrixclouddaaschecks/ 
please test and give a feedback @samaanbag
